### PR TITLE
Support Debian Bullseye (testing)

### PR DESCRIPTION
**Pull request by joshuaboniface**

### CODE REVIEW

**Review by Artiume — APPROVED** [review]
Cool, I'll start testing docker arm.

What is gcc version?

**Comment by joshuaboniface** [comment]
@Artiume I picked 8 since it was available, but that is flexible.

**Comment by Artiume** [comment]
I'll read through the build scripts. I wasn't sure what it was for, but I think I have an understanding now.

**Review by andrewrabert — APPROVED** [review]

File: build.yaml [hunk 1]
```diff
@@ -1,14 +1,17 @@
 ---
 # We just wrap `build` so this is really it
 name: "jellyfin-ffmpeg"
-version: "4.2.1-2"
+version: "4.2.1-4"
```
Comment (andrewrabert) [inline build.yaml:4]: curious - where did 3 go?

Comment (joshuaboniface) [reply build.yaml:4]: It existed, but we never updated the build.yaml - doesn't seem to have mattered though.

**Pull request merged**